Update README.md (#101)
vtex-apps/minicart: README.md

This app uses our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)

> This app is used as a block in our [Store Header](https://github.com/vtex-apps/store-header/blob/master/store/interfaces.json). You can configure the minicart properties 

Reviewer: Looks like here it's missing something. `You can configure the minicart properties`. How? Where?

> To configure or customize this app, you need to import in your dependencies on `manifest.json`.

Reviewer 2: ```suggestion
To configure or customize this app, you need to import it in your dependencies in `manifest.json`.
```

```json
  dependencies: {
    "vtex.minicart": "2.x"
  }
```

> Then, add `minicart` block into your app theme as we do in our [Dreamstore app](https://github.com/vtex-apps/dreamstore/blob/master/store/blocks.json). 

Reviewer: Why there is the `Store Header` example and the `Dreamstore` one?

Author: Store Header is the app that adds minicart block. Dreamstore is where we configure the minicart behavior.

Now, you can change the behavior of the minicart block that is in the store header. See an example of how to configure: 
```json
"minicart": {
  "blocks": [
    "product-summary"
  ],
  "props": {
    "type": "popup",
    "showRemoveButton": true,
    "showDiscount": true,
    "showSku": true,
    "labelMiniCartEmpty": "",
    "labelButtonFinishShopping": "Ir para o checkout",
    "enableQuantitySelector": true,
    "maxQuantity": 10
  }
}
```
### Blocks API

> Also, is important to understand ho are the constraints of the minicart interface that must be implemented by a block when you want to use the minicart.

Reviewer 3: "ho"?

> Also, is important to understand which are the constraints of the minicart interface that must be implemented by a block when you want to use the minicart.

Reviewer 2: ```suggestion
When implementing this app as a block, various inner blocks may be available. The following interface lists the available blocks within minicart and describes if they are required or optional.
```

  }
}
```

> The minicart has as a required block the `product-summary`. So, any minicart block implementation created must add a product-summary as a block that is inside of minicart. To know how is the API of `product-summary` see the next section.

Reviewer 2: ```suggestion
The minicart has as a required block the `product-summary`. So, any minicart block implementation created must add a product-summary as a block that is inside of minicart. (Similarly, `product-summary` has its own inner block structure that can be configured. There is a link to its API in the next section.)
```

#### Configuration 
Through the Storefront, you can change the minicart's behavior and interface. However, you also can make in your theme app, as Dreamstore does.

> | Prop name                   | Type       | Description                                                        |

Reviewer 4: I think it is important to add a `Default value` field in the configuration table.
e.g.

```
| Prop name | Type | Description  | Default value
| ------------------ | ---------- | ---------------------- |
| `type` | `String`| Define Minicart mode. (values: 'popup' or 'sidebar')| `popup`
```

This app provides some CSS classes as an API for style customization.

Class name        | Description                    | Component Source

> ----------------- | ------------------------------ | ------------------------

Reviewer 3: (I think "indenting" this just makes it more difficult to add or remove an item of the table, because you always have to align with the wider column.)

Author: Yeah, I agree. Is just my TOC. :developers:

  margin-top: 10px;
}
```
#### CSS namespaces

> Below, we describe the namespaces that is define in the minicart.

Reviewer 4: that are defined in the minicart
